src/controllers/currentWarningsController.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 const mongoose = require('mongoose');
 const userProfile = require('../models/userProfile');
-
+const helper = require('../utilities/permissions');
```

Comment:
```suggestion
const {hasPermission} = require('../utilities/permissions');
```
I usually import it destructured like this to make the calls shorter (hasPermission vs helper.hasPermission), but don't think it really matters. Up to you.